src/components/app.js

```diff
@@ -141,6 +145,12 @@ const App = () => {
 
     const [tabIndex, setTabIndex] = useState(0);
 
+    const [maintenanceSnackBarId, setMaintenanceSnackBarId] = useState(null);
```

Comment:
what happend if we recieve a second message before the first one expire ?

Reply:
now the previous msg is cancelled when receiving a new one